dojo_plugin/pages/settings.py

```diff
@@ -17,8 +17,8 @@ def settings_override():
     user = get_current_user()
     tokens = UserTokens.query.filter_by(user_id=user.id).all()
 
-    ssh_key = SSHKeys.query.filter_by(user_id=user.id).first()
-    ssh_key = ssh_key.value if ssh_key else None
+    ssh_keys = SSHKeys.query.filter_by(user_id=user.id).all()
```

Comment:
`SSHKeys.query.filter_by(user_id=user.id).with_entities(SSHKeys.value).all()`
Minor, but figured I'd mention it; this gets you the data you want.